### ISSUE TITLE
Unable to complete the Onfido check on Android browsers

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#5299]
CC @MitchExpensify 
___

NOTE: We have recent reports with more info (for example, using the Brave browser), please can [these](https://github.com/Expensify/App/issues/5299#issuecomment-2433771547) be [read](https://github.com/Expensify/App/issues/5299#issuecomment-2366772760) before testing, thanks!

## Action Performed:
- Create a workspace
- Tap 'Get Started'
- Add a bank acocunt, using credentials 1 from [the StackOverflow guide](https://stackoverflow.com/c/expensify/a/525/297)
- Follow the steps until you reach the Onfido check
- Use an Android device to scan the QR code and begin the Onfido flow
- Select any verification type
- Scan the front and back of the verification ID

**User report:** 

> Trying to use onfido to verify identity and Camera and Microphone are fully allowed in browser and in device settings.
Every time I click Enable Both for Allow Camera and Microphone Access, it pops up with Camera and microphone access denied.
> Tried with 3 browsers (Firefox Beta, Chrome, Samsung Browser) on Pixel 8 Pro running Android 14.

## Expected Result:
- You are then not prompted to enable the camera permission

## Actual Result:
- You are then prompted to enable camera permission -- clearly it is already enabled though!

https://user-images.githubusercontent.com/10736861/134701301-4074f2a2-2c42-4c97-82ea-34bbf691bbd9.mp4

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

Users cannot add bank accounts for payment

## Platform:

Where is this issue occurring?

- Web ❌ 
- iOS ❌ 
- Android ✅ 
- Desktop App ❌
- Mobile Web ❌

**Version Number:**
**Reproducible in staging?:** Yes
**Reproducible in production?:** N/A
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:**  See above
**Expensify/Expensify Issue URL:** N/A
**Issue reported by:** @Julesssss 
**Slack conversation:** https://expensify.slack.com/archives/C020EPP9B9A/p1631806430343700?thread_ts=1631803999.337800&cid=C020EPP9B9A

**Comment by MelvinBot** [comment]
Triggered auto assignment to @mateocole (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by Julesssss** [comment]
This is on Onfido's side, I think we need to confirm, then reach out to them and raise a bug.

**Comment by Julesssss** [comment]
CC @kevinksullivan, maybe I should have created this as an Internal issue?

**Comment by Julesssss** [comment]
The Onfido SDK doesn't have any issues which match my bug, so I'm going to record the failure and open a new issue.

**Comment by MelvinBot** [comment]
@Julesssss, @mateocole Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by mateocole** [comment]
@Julesssss would this issue be used to track the status and update info as we receive it from Onfido? Or, is there actions a member of engineering should be assigned to this to follow up on?

**Comment by mateocole** [comment]
Okay sounds good!

**Comment by Julesssss** [comment]
[Issue](https://github.com/onfido/react-native-sdk/issues/45). Awaiting response from the Onfido team.

**Comment by ctkochan22** [comment]
@Julesssss can you test if you are able to complete this via the old dot way.
1. Go to payments on **old dot**
2. Create a verified BBA
3. Go to onfido
4. Scan the QR code

**Comment by Julesssss** [comment]
Good idea @ctkochan22.

I can confirm that I'm seeing the same issue here, which means this shouldn't be considered a blocker.

**Comment by Julesssss** [comment]
Separately, I have been able to verify that the mWeb/QR code flow works on an older Android device (Nexus 5X) using Chrome. Something about my Pixel 2's configuration must be causing this issue, odd.

**Comment by ctkochan22** [comment]
This is on staging or prod? I can fire up my dev to check it out

**Comment by Julesssss** [comment]
It occurs in all environments, and I don't think it's a bug in our implementation. It's the id.onfido.com webpage that is launched via a QR code if you attempt to add a card on web. When I switch to Android I'm stuck in this permission loop.

**Comment by Julesssss** [comment]
This seems to be a rare issue, so I'm reducing the priority.

**Comment by AluminumHaste** [comment]
This is STILL broken on Android.
Trying to use onfido to verify identity and Camera and Microphone are fully allowed in browser and in device settings.
Every time I click Enable Both for Allow Camera and Microphone Access, it pops up with Camera and microphone access denied.

Tried with 3 browsers (Firefox Beta, Chrome, Samsung Browser) on Pixel 8 Pro running Android 14.

**Comment by Julesssss** [comment]
Hi @AluminumHaste, thanks for sharing. Lets see if QA are able to reproduce.

**Comment by Julesssss** [comment]
Waiting on retest

**Comment by dangerweenie** [comment]
also happening on brave browser for Android - camera is enabled in brave app permissions as well as in "site settings",  but I am not getting any pop-up or prompt - it's just saying camera access is not allowed.

**Comment by kavimuru** [comment]
Issue not reproducible

https://github.com/user-attachments/assets/63ff91d2-c3a0-4f7a-9594-288266dd9461

**Comment by Julesssss** [comment]
For the next retest, would it be possible to retest this issue on the Android Brave browser, please? It sounds like customers can reproduce in certain circumstances

**Comment by Julesssss** [comment]
Awaiting retest

**Comment by Julesssss** [comment]
I requested a retest manually [here](https://expensify.slack.com/archives/C9YU7BX5M/p1732703998002499)

**Comment by izarutskaya** [comment]
Issue not reproducible 
Galaxy M 52 5G/ Android 13

https://github.com/user-attachments/assets/2032108f-4217-4cd1-aa68-1c8f65a356e1
